File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcode/AdvanceBarcodeFeatures/PatchCode.php
<?php
namespace Aspose\Barcode\WorkingWithBarcode\AdvanceBarcodeFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;

class PatchCode{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $builder = new BarCodeBuilder();

        # Set Symbology type
        $symbology=new Symbology();
        $builder->setSymbologyType($symbology->PatchCode);

        # Set code text
        $builder->setCodeText("Patch T");

        # Save the image to your system and set its image format to Jpeg
        $builder->save($dataDir . "PatchCode.jpg");

        # Display Status
        print "Generated PatchCode Successfully.".PHP_EOL;

    }

}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWithBarcode/AdvanceFeatures/PatchCode.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology

class PatchCode:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithBarcode/AdvanceBarcodeFeatures/PatchCode/'
        
       # Instantiate barcode object
        builder = BarCodeBuilder()

        # Set Symbology type
        symbology = Symbology
        builder.setSymbologyType(symbology.PatchCode)

        # Set code text
        builder.setCodeText("Patch T")

        # Save the image to your system and set its image format to Jpeg
        builder.save(dataDir + "PatchCode.jpg")

        # Display Status
        print "Generated PatchCode Successfully."

if __name__ == '__main__':        
    PatchCode()